EdvClient.js

```diff
@@ -585,9 +585,9 @@ export class EdvClient {
     if(count === true) {
       return response.data;
     }
-    const docs = response.data;
+    const {data: {documents}} = response;
 
-    return Promise.all(docs.map(
+    return Promise.all(documents.map(
       encryptedDoc => this._decrypt({encryptedDoc, keyAgreementKey})));
```

Comment:
@dlongley were you looking for the return of the `edvClient.find` API here to be `[...]` or `{documents: [...]}`

Reply:
I think `{documents: [...]}` would be better to leave room for other things.